content/game_process.tex

```diff
@@ -100,33 +100,32 @@ \subsubsection{Invalid Goal}
 
 A goal is invalid (that is, it can never be awarded) in the following circumstances:
 \begin{enumerate}
-    \item When an Indirect Kick has not occurred (see~\cref{sec:indirect_kick}).
+    \item When an Indirect Kick is required has not occurred (see~\cref{sec:indirect_kick}).
```

Comment:
[...] is required *but* has not occured?
[...] *that* required has not occured?

Reply:
[...] is required but has not occurred

Reply:
Done